app/src/main/java/org/flyve/mdm/agent/EditUserActivity.java

```diff
@@ -17,10 +24,15 @@
 
 import org.flyve.mdm.agent.core.user.UserController;
```

Comment:
This would be fine on a library for core, now when I copy this package to dashboard will work great and order in the same package